README.md

```diff
@@ -0,0 +1,55 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+Description the project.
```

Comment:
- [ ] Kindly write a description of your project, this will make your readme professional 👍🏽